developer_guide/installation/installation_archive.rst

```diff
@@ -0,0 +1,123 @@
+Installing Akeneo PIM with the Archive
+======================================
+
+This document provides step by step instructions to install the PIM with the archive on an environment that fulfill the expected :doc:`system_requirements/system_requirements`.
+
+Getting Akeneo PIM
+------------------
+
+Downloading the archive
+***********************
+
+From http://www.akeneo.com/download you can choose to download a PIM edition with (called *icecat*) or without (called *minimal*) demo data. If you prefer, you can also download them directly from the command line:
+
+.. code-block:: bash
+    :linenos:
+
+    $ wget http://download.akeneo.com/pim-community-standard-v1.5-latest-icecat.tar.gz #for icecat version
+    $ wget http://download.akeneo.com/pim-community-standard-v1.5-latest.tar.gz #for minimal version
+
+Extracting the archive
+**********************
+
+.. code-block:: bash
+    :linenos:
+
+    $ mkdir -p /path/to/installation
+    $ tar -xvzf pim-community-standard-v1.5-latest-icecat.tar.gz -C /path/to/installation
+    $ cd /path/to/installation
+
+.. note::
+    Replace */path/to/installation* by the path to the directory where you want to install the PIM.
+
+.. note::
+    Replace *pim-community-standard-v1.5-latest-icecat.tar.gz* by the location and the name of the archive
+    you have downloaded from http://www.akeneo.com/download.
+
+
+Initializing Akeneo
+-------------------
+.. code-block:: bash
+    :linenos:
+
+    $ cd /path/to/installation/pim-community-standard
+    $ php app/console cache:clear --env=prod
+    $ php app/console pim:install --env=prod
+
+
+Configuring the virtual host
+----------------------------
+
+Enabling Apache mod_rewrite
+***************************
+
+.. code-block:: bash
+    :linenos:
+
+    $ sudo a2enmod rewrite
+
+Creating the vhost file
+***********************
+
+Create the file ``/etc/apache2/sites-available/akeneo-pim.local.conf``
+
+.. code-block:: apache
+    :linenos:
+
+    <VirtualHost *:80>
+        ServerName akeneo-pim.local
+
+        DocumentRoot /path/to/installation/pim-community-standard/web/
+        <Directory /path/to/installation/pim-community-standard/web/>
+            Options Indexes FollowSymLinks MultiViews
+            AllowOverride All
+            Require all granted
+        </Directory>
+        ErrorLog ${APACHE_LOG_DIR}/akeneo-pim_error.log
+
+        LogLevel warn
+        CustomLog ${APACHE_LOG_DIR}/akeneo-pim_access.log combined
+    </VirtualHost>
+
+Enabling the virtualhost
+************************
+
+.. code-block:: bash
+    :linenos:
+
+    $ sudo a2ensite akeneo-pim.local
+    $ sudo apache2ctl -t
```

Comment:
apache2ctl is not the right way to do it, you will not execute system specific part. service is more standard